lib/conjur/id.rb

```diff
@@ -18,10 +18,13 @@
 # IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
+require 'conjur/escape'
 
 module Conjur
   # Encapsulates a Conjur id, which consists of account, kind, and identifier.
   class Id
+    extend Conjur::Escape::ClassMethods
```

Comment:
If you `include Conjur::Escape::ClassMethods`, you'll be able to call the methods directly, rather than having to use `self.class` to get to them.